src/me/actions/thunks/index.ts

```diff
@@ -28,12 +28,27 @@ export const getMe = () => async (
   getState: GetState
 ) => {
   try {
-    const resp = await apiGetApiMe({})
+    let user
 
-    if (resp.status !== 200) {
-      throw new Error(resp.data.message)
+    if (
+      isFlagEnabled('multiAccount') &&
+      isFlagEnabled('avatarNameForMultiAccountFix')
+    ) {
+      const resp = await getAccounts({})
+
+      if (resp.status !== 200) {
+        throw new Error(resp.data.message)
+      }
+      user = resp.data.filter(account => account.isActive)[0]
```

Comment:
i think [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) is made specifically for this.

note, if for some reason there's no active account at this stage, this might cause errors / crashes since code below this relies on `user` existing and having an `id` on it

Reply:
Just pushed the `find` fix. 

For the issue of none of the account being active, there is an assumption throughout that atleast one account will be active. 

This comment confirms that: https://github.com/influxdata/ui/blob/master/src/accounts/AccountPage.tsx#L60-L68